modules/govuk/manifests/node/s_api_lb.pp

```diff
@@ -60,6 +61,14 @@
     internal_only => true,
   }
 
+  if !empty($mapit_servers) {
+    loadbalancer::balance { 'mapit':
+      servers       => $mapit_servers,
+      internal_only => true,
+      https_only    => false, # FIXME: Remove for #51136581
```

Comment:
I'd like to, but given that we're testing the apps for compatibility with new MapIt, I wanted to keep the other variables to a minimum.  I'd envisaged coming back to this later on.  (Not that I've got any idea what 51136581 refers to)